cobc/ppparse.y

```diff
@@ -1368,6 +1373,33 @@ leap_second_directive:
 | OFF
 ;
 
+imp_directive:
+  /* GnuCOBOL 3.3 extension */
+  INCLUDE
+  {
+	ppparse_error (_("Missing argument for IMP INCLUDE directive"));
```

Comment:
I'm not sure if this was what you expected. I only get this error when arguments are missing and not others. You talked about "resetting the error".

Reply:
I've meant to let the parser do the error output, but catch that (so it doesn't fall through and consumes more):
instead of INCLUDE and nothing (=manually catched) you do `INCLUDE error` and do an yyerrok call (see existing examples).
As the directive is `>> IMP` and we only support one of those per line, it may be more reasonable to do that kind of error handling there -> just directly a single `error`.

You should be able to tinker with that, seeing what the difference in the testsuite it makes.